docs/configuration.md

```diff
@@ -136,6 +144,26 @@ spec:
     targetPort: 53
 ```
 
+## Fine-Tuning Load Balancing Methods with NGINX Plus
```

Comment:
this section is inserted in the middle of `TCP/UDP Load Balancing Extension`. Please move it